daemon/mqtt/matrix-mqtt

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
```

Comment:
Even if we missed this so far -- please add a license header when creating new files. I'll amend this in existing files soon…